src/helpers.c

```diff
@@ -538,7 +538,7 @@ enum swupd_code swupd_init(void)
 	/* Check that our system time is reasonably valid before continuing,
 	 * or the certificate verification will fail with invalid time */
 	if (timecheck) {
-		if (!verify_time()) {
+		if (!verify_time(path_prefix)) {
```

Comment:
if it fails, use verify_time(NULL). 

In the case we are doing an installation to an empty directory using swupd verify --install, we won't have a valid timestamp in path_prefix.

Reply:
ok, I'll make that change